recipes/tests.rb

```diff
@@ -208,11 +208,22 @@
 # Amazon Time Sync
 ###################
 if node['init_package'] == 'init'
-  chrony_check_command = "service #{node['cfncluster']['chrony']['service']} status | grep -i running"
+  get_chrony_status_command = "service #{node['cfncluster']['chrony']['service']} status"
 elsif node['init_package'] == 'systemd'
   # $ systemctl show -p SubState <service>
   # SubState=Running
-  chrony_check_command = "systemctl show -p SubState #{node['cfncluster']['chrony']['service']} | grep -i running"
+  get_chrony_status_command = "systemctl show -p SubState #{node['cfncluster']['chrony']['service']}"
+end
+chrony_check_command = get_chrony_status_command + " | grep -i running"
+
+ruby_block 'log_chrony_status' do
+  block do
+    get_chrony_service_log_command = "journalctl -u #{node['cfncluster']['chrony']['service']}"
```

Comment:
is journalctl systemd-specific? [the man page](https://www.freedesktop.org/software/systemd/man/journalctl.html) makes me think it is. if so will this break on initd systems?